src/globus_cli/commands/flows/_common.py

```diff
@@ -115,6 +115,32 @@
     """,
 )
 
+run_managers_option = click.option(
+    "--run-manager",
+    "run_managers",
+    type=str,
+    multiple=True,
+    help="""
+        A principal that may manage the flow's runs.
+
+        This option can be specified multiple times
+        to create a list of run managers.
+    """,
+)
+
+run_monitors_option = click.option(
+    "--run-monitor",
+    "run_monitors",
+    type=str,
+    multiple=True,
+    help="""
+        A principal that may monitor the flow's runs.
+
+        This option can be specified multiple times
+        to create a list of run monitors.
+    """,
+)
+
```

Comment:
These are only used in the `create` command right now, correct?

I would prefer that we not put them into this `_common` module and instead declare them inline on the appropriate command.

The `_common` modules (there are several, for different families of commands) are for things which are being shared across commands.
Putting things here gives the false impression that we have a kind of uniformity between `create` and `update` which we don't have.

Fixing the divergence between those two commands is clearly beyond the scope of this PR, but we don't need to continue to declare things in a way which may mislead us in the future into not understanding how much of our code is _not_ being reused across components.

Reply:
@sirosen This should be resolved now along with the changelog item.